File: sparktrain/src/main/scala/com/xtb/spark/domain/CourseClickCount.scala
package com.xtb.spark.domain

/**
  * 实战课程的点击数实体类
  * @param day_course   对应的就是HBase的rowKey --> 20180502_130
  * @param click_count  对应的20180502_130的访问总数
  */
case class CourseClickCount(day_course: String, click_count: Long)

File: spark-web/src/main/java/com/xtb/spark/sparkweb/domain/CourseClickCount.java
package com.xtb.spark.sparkweb.domain;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

/**
 * 实战课程访问量实体类
 */
@Data
@NoArgsConstructor(staticName = "of")
@Accessors(chain = true)
public class CourseClickCount {
    private String name;
    private long value;
}
